settings.yml

```diff
@@ -141,7 +133,7 @@ big_picture:
     name: Landscape
     short_name: Land...
     title: Interactive Mobility Landscape
-    fullscreen_header: Overwhelmed? Please see the CNCF Trail Map. That and the interactive landscape are at l.cncf.io
+    fullscreen_header: Overwhelmed? Please see the Trail Map. 
```

Comment:
nit: we don't have a trail map; we should change this to sth like "missing data? submit a pull request to add entries"